.github/workflows/lint.yaml

```diff
@@ -0,0 +1,23 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: conda-incubator/setup-miniconda@v2
+
+    - name: Install packages
+      run: |
+        conda env create --name test-env -f environment-dev.yml
```

Comment:
Slight inconsistency here between the environment called `test` vs `dev`.

Reply:
Same: `environment-dev.yml` will have a project-specific name; and I wanted the GitHub workflow to be generic enough to be able to copy it around, that's why a generic environment name.